include/mls/messages.h

```diff
@@ -95,6 +95,12 @@ struct PreSharedKeys
   TLS_TRAITS(tls::vector<2>)
 };
 
+struct PSKWithSecret
+{
+  PreSharedKeyID id;
+  bytes secret;
+};
```

Comment:
do we need serialization info for this struct ?

Reply:
No, it's just internal.  We're not going to send the secret around :)